### DISCUSSION TITLE
"loudness" button / change curve depending on volume?

### DISCUSSION THREAD

**Discussion by jw-dsgn** [bitgapp/eqMac#652] [Ideas]
I find myself primarily using eqMac to boost highs and lows at low volumes (basically serving the same purpose as the loudness button found on old hifi receivers) but at higher volumes my settings become a bit much. I get around this by having a few presets that I switch to depending on volume. It would be awesome if this could be automated somehow.

Idk how feasible this is, but three ways I can think of doing it:

1. A dry/wet mix knob
2. Old-timey loudness button/checkbox that just boosts highs and lows if volume is below a certain threshold
3. My dream come true... if either the dry/wet mix or the curve itself could change based on volume, e.g. automatically 100% wet at ~10% volume and 0% at ~90% volume

Anyway, just a thought. Thanks, love the app!

**Comment by mustevenplay** [comment]
Is there any vst/au plugin that does this already? Just out of curiosity
